src/traces/waterfall/attributes.js

```diff
@@ -0,0 +1,108 @@
+/**
+* Copyright 2012-2019, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var barAttrs = require('../bar/attributes');
+var lineAttrs = require('../scatter/attributes').line;
+
+function directionAttrs() {
+    return {
+        editType: 'style',
+        color: barAttrs.marker.color,
+        opacity: barAttrs.marker.opacity,
+        line: {
+            editType: 'style',
+            color: barAttrs.marker.line.color,
+            width: barAttrs.marker.line.width
+        }
+    };
+}
+
+module.exports = {
+
+    valuetype: {
```

Comment:
I'm not a big fan of `valuetype` which to me sounds like a scalar. I'm voting for `sums` but I have no strong opinion.

Reply:
(copying @nicolaskruchten's comment so I can see the discussion alongside the code)
> I agree that `valuetype` isn't great, but `sums` doesn't seem right either... "sum = relative", "sum = total" ?

How about `measure`? Generic enough that people would need to read the docs to understand its meaning, but correspondingly free of baggage.

Is there even a use case for this being a scalar? I'd think this should be a `data_array` - also that `dflt` should be omitted and the expected values listed in `description`. I guess the one case I could imagine wanting to make easier than having to provide a whole array is putting a single sum at the end, so we could allow a scalar to mean "let all but the last one be relative, and do this (absolute or total) with the last one."

Reply:
I'm a fan of `measure` :+1: